app/liquid_tags/organization_tag.rb

```diff
@@ -2,7 +2,12 @@ class OrganizationTag < LiquidTagBase
   include ApplicationHelper
   include ActionView::Helpers::TagHelper
   PARTIAL = "organizations/liquid".freeze
-  REGISTRY_REGEXP = %r{#{URL.url}/(?<org_slug>[\w-]+)(?:/)?(?:[\w-]+)?}
+
+  # @todo What if someone provides https://dev.to/terms-of-service.  That will meet this criteria.
+  #       How do we want to consider handling that situation?  My assumption is that we might want
+  #       to test if the org_slug is a valid organization before we even let the OrganizationTag say
+  #       there's a match.
+  REGISTRY_REGEXP = %r{#{URL.url}/(?<org_slug>[\w-]+)/?}
```

Comment:
thank you!

re: todo. The tag will raise an error if org not found (farther down) - and if you're linking to t-o-s, you probably want to use a link (not an embed). Not sure this fully covers the concern, but I think the risk of confusion/conflict is pretty low (I'd be more worried we'd mistakenly match a tag or user than a page)

```ruby
     organization = Organization.find_by(slug: organization)
     raise StandardError if organization.nil?
```

Reply:
I think what I'm concerned about is when we start looking at other embed's on Forem.  Which Tag will "win" on a match?

This is certainly a boundary condition and doesn't detract for the current work.